frontend/src/components/lists/styles/data.scss

```diff
@@ -45,3 +49,7 @@
         }
     }
 }
+
+.dragging-item {
+    box-shadow: 3px 3px 3px 1px gray
```

Comment:
For consistency, I try to add all the colours in `styles/_include.scss`. Could you add the colour there or use an existing one?

Reply:
I hadn't spotted the existing `shadow` mixin, I've used that